src/components/caniuse-feature.astro

```diff
@@ -10,20 +16,66 @@ const browsers = {
 	'and_ff': 'Firefox for Android',
 	'and_qq': 'QQ Browserr',
 };
+
+// Similar order as MDN
+const browser_order = [
+	// Desktop
+	"chrome",
+	"edge",
+	"firefox",
+	"ie",
+	"opera",
+	"safari",
+	// Mobile
+	"and_chr",
+	"and_ff",
+	"op_mob",
+	"ios_saf",
+	"samsung",
+	"android",
+];
+
+const feature_browsers = Object.keys(feature.browser_support);
+feature_browsers.sort((a, b) => {
+	const a_index = browser_order.indexOf(a);
+	const b_index = browser_order.indexOf(b);
+	if (a_index === -1) {
+		return 1;
+	}
+	if (b_index === -1) {
+		return -1;
+	}
+	return a_index - b_index;
+});
+
 ---
-{ feature.browser_support && Object.keys(feature.browser_support).length > 0 && (
+{ link && feature.browser_support && feature_browsers.length > 0 && (
 <a
 		class="cssdb-feature-caniuse"
 		href={link}
 		target="_blank"
 		rel="noreferrer"
-		aria-label={`can i use browser support for ${title}`}
+		aria-label={`View the browser support for ${title}. Opens in a new tab.`}
 >
-	{Object.keys(feature.browser_support).map(id => (
+	{feature_browsers.map(id => (
 		<span class={`cssdb-browser cssdb-browser--${id}`}>
 		 <span class="sr-only">Supported on {browsers[id] || id} since version </span>
 		 <span class="cssdb-browser-version">{feature.browser_support[id]}</span>
 		</span>
 	))}
 </a>
 ) }
+
+{ !link && feature.browser_support && feature_browsers.length > 0 && (
```

Comment:
When there is no link at all we still want to show stats if we have them.

A good example of this is double position stops in gradient functions.
This covers multiple gradient functions and doesn't have a single link to docs.